Source/ImPlotWidgets/Public/LineFilledGraphWidget.h

```diff
@@ -5,12 +5,18 @@
 #include <ImGui/Public/Widgets/ImGuiCustomWidget.h>
 #include <Runtime/SlateCore/Public/Widgets/SCompoundWidget.h>
 
+struct FDataPoint
+{
+	float X, Y;
```

Comment:
```suggestion
	float X = 0.0f;
	float Y = 0.0f;
```

I'm fairly sure not declaring things on the same line is a UE standards thing, anyway it can become really confusing fast if you're declaring anything more than a primitive data type. Defaulting member vars to something is generally sensible too.